sprint8/tema2.js

```diff
@@ -0,0 +1,13 @@
+function sumArray(numbers) {
```

Comment:
```suggestion
function getSumArray(numbers) {
```

un nume mai potrivit ar fi un prefix "get" care ne da senzatia din numele fisierului ca functia va returna ceva